### PR TITLE
minimal brew cask file

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Signed-off-by: Pierre-Emmanuel Jacquier <pierre-emmanuel.jacquier@epitech.eu>

### CODE REVIEW

**Comment by pierre-emmanuelJ** [comment]
https://travis-ci.org/Homebrew/homebrew-cask/builds/394634017?utm_source=github_status&utm_medium=notification

**Comment by pierre-emmanuelJ** [comment]
try

`brew cask style Casks/exo.rb`

**Comment by greut** [comment]
mmmh https://goreleaser.com/customization/#Homebrew

**Comment by pierre-emmanuelJ** [comment]
ooohhww thx

File: .goreleaser.yml [hunk 1]
```diff
@@ -49,3 +49,30 @@ nfpm:
...
+    name: pierre-emmanuelJ
+    email: pierre-emmanuel.jacquier@epitech.eu
+
+  # Your app's homepage.
+  # Default is empty.
+  homepage: "https://github.com/exoscale/egoscale/tree/master/cmd/exo"
+
+  # Your app's description.
+  # Default is empty.
+  description: "Manage easily your Exoscale infrastructure from the exo command-line"
```
Comment (greut) [inline .goreleaser.yml]: which one did you mean?

- _from the command-line_
- _with the exo command-line_

File: .goreleaser.yml [hunk 2]
```diff
@@ -49,3 +49,35 @@ nfpm:
   formats:
     - deb
     - rpm
+
+brew:
+  name: exo
+
+  github:
+    owner: pierre-emmanuelJ
```
Comment (nfrankel) [inline .goreleaser.yml]: Shouldn't the owner be the exoscale org?

File: cmd/exo/exo.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+cask 'exo' do
+  version '0.10.0a4'
```
Comment (nfrankel) [inline cmd/exo/exo.rb]: Where does the version come from? Shouldn't it be dynamic?

File: cmd/exo/exo.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 '30943239ce174dea5399ae0a192617b44b287c0fb874b27d4bf55267d82095cd'
```
Comment (nfrankel) [inline cmd/exo/exo.rb]: sha should be dynamic as well

File: cmd/exo/exo.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+  url "https://github.com/exoscale/egoscale/releases/download/v#{version}/egoscale_#{version}_darwin_amd64.tar.gz"
+  name 'EXO'
```
Comment (nfrankel) [inline cmd/exo/exo.rb]: Why the uppercase?

**Comment by nfrankel** [comment]
I believe the Ruby file should be generated along the other artifacts during the build. I don't know _how_ to do it though :-)

File: .goreleaser.yml (hunk 2 above)
Comment (pierre-emmanuelJ) [reply .goreleaser.yml]: It's was WIP

File: cmd/exo/exo.rb (hunk 3 above)
Comment (pierre-emmanuelJ) [reply cmd/exo/exo.rb]: It's was WIP (will be done with goreleaser)

File: cmd/exo/exo.rb (hunk 4 above)
Comment (pierre-emmanuelJ) [reply cmd/exo/exo.rb]: idem

**Comment by pierre-emmanuelJ** [comment]
Thanks for your review

**Pull request closed without merging**